automatic/keeweb/keeweb.nuspec

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek
+     uppercase omega letter enclosed in quotation marks, you should use an
+     editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+    <metadata>
+        <id>keeweb</id>
+        <title>KeeWeb</title>
+        <version>1.6.2</version>
+        <authors>antelle</authors>
+        <owners>wget</owners>
+        <summary>Free cross-platform password manager compatible with KeePass</summary>
+        <description>This webapp is a browser and desktop password manager compatible with KeePass databases. It doesn't require any server or additional resources. The app can run either in browser, or as a desktop app.</description>
+        <projectUrl>https://keeweb.info/</projectUrl>
+        <tags>keeweb keypass password admin</tags>
```

Comment:
missing madatory tags: `foss` `cross-platform`

Reply:
@majkinetor You mean the final nupkg package should have both (32 and 64bits) installers bundled? That's 40Mio *2 for both :-/

Reply:
Yes, because it makes package stable and the one you can depend on. 
There are number of other packages like that here, they are marked with special icon in report. There is also an [open issue](https://github.com/chocolatey/chocolatey-coreteampackages/issues/866) about this.

Reply:
Ok I updated the package according to the changes you requested.